lib/features/ai_api_scan/services/food/food_text_analysis_service.dart

```diff
@@ -228,16 +232,17 @@ class FoodTextAnalysisService extends BaseGeminiService {
         Ingredients: ${food['ingredients']}
         Similarity Score: ${(food['score'] * 100).toStringAsFixed(1)}%
         '''));
-        
-        if (food['image_url'] != null && food['image_url'].toString().isNotEmpty) {           
-              if (refIndex == 1) {
-                selectedImageUrl = food['image_url'];
-              }
+
+        if (food['image_url'] != null &&
+            food['image_url'].toString().isNotEmpty) {
+          if (refIndex == 1) {
+            selectedImageUrl = food['image_url'];
+          }
```

Comment:
```suggestion
        if (food['image_url'] != null &&
        food['image_url'].toString().isNotEmpty &&
        refIndex == 1) {
        selectedImageUrl = food['image_url'];
    }
```

this can be improved like this to improve readibility